deploy/helm/kubecf/assets/operations/instance_groups/app_autoscaler.yaml

```diff
@@ -0,0 +1,1150 @@
+  value: ((servicebroker_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asapi/jobs/name=servicebroker/properties/autoscaler/service_broker/public_ca_cert?
+  value: ((servicebroker_public_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=scalingengine/properties/autoscaler/policy_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=scalingengine/properties/autoscaler/scalingengine_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=scalingengine/properties/autoscaler/scheduler_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=scheduler/properties/autoscaler/policy_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=scheduler/properties/autoscaler/scheduler_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=operator/properties/autoscaler/policy_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=operator/properties/autoscaler/appmetrics_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=operator/properties/autoscaler/instancemetrics_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=operator/properties/autoscaler/scalingengine_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=operator/properties/autoscaler/lock_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asmetrics/jobs/name=metricsserver/properties/autoscaler/instancemetrics_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asmetrics/jobs/name=metricsserver/properties/autoscaler/policy_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asmetrics/jobs/name=eventgenerator/properties/autoscaler/appmetrics_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asmetrics/jobs/name=eventgenerator/properties/autoscaler/policy_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asnozzle/jobs/name=metricsgateway/properties/autoscaler/policy_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asapi/jobs/name=apiserver/properties/autoscaler/policy_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+- type: replace
+  path: /instance_groups/name=asapi/jobs/name=servicebroker/properties/autoscaler/binding_db/tls/ca?
+  value:  ((postgres_ca.certificate))
+
+- type: replace
```

Comment:
Why can't we make this working with SSL validation?

Reply:
The API `https://api. ((system_domain))` is provided by the CF deployment . If the related cert is  self-signed, the `ssl_skip_validation` should be set `true`.

Reply:
Could you leave a `TODO` here like:
```
TODO: make these skip_ssl_validation conditional if the router cert is provided by the user.
```

Reply:
Added the TODO.